### ISSUE TITLE
Downgrade 1.20.1 Forge version from 47.2.0 to 47.1.3

### ISSUE THREAD

**Issue by NightrainsRbx** [tweaked-programs/cccbridge#81]
The recently updated forge port for 1.20.1 requires forge 47.2.0, most modders are staying on 47.1.3 until NeoForge becomes viable enough, this is due to internal conflicts in the Forge team.

Downgrading to 47.1.3 would bring better compatibility with modpacks.

It would only need to change the version in the gradle properties, no code changes are required.

**Comment by SammyForReal** [comment]
Thanks for the issue.
Yeah I didn't think about that, I do not want this mod to be missing out for people in modpacks because of that.

Will try to recompile with said version or similar.

**Comment by NightrainsRbx** [comment]
Thanks!

**Comment by virtualsaturn** [comment]
Yeah, the current forge version crashes any version of neoforge on launch. Unsure why, most other mods that ask for 47.2.0+ will throw a version mismatch error but still function correctly.

**Comment by GGORG0** [comment]
Hi, are there any news on that 47.1 build? The latest version of NeoForge for 1.20.1 currently is 47.1.104...

**Comment by Multidakman** [comment]
This is still a problem where is the update, it has been like half a year now, plus NeoForge 1.20.1 version only goes up to version 47.1.106

**Comment by SammyForReal** [comment]
It was a busy year for me. 
PRs welcome

**Comment by SammyForReal** [comment]
Fixed with bc189087ccb168056fe3bb893683bbd69659d56d
Will be released with the next update
